lib/matplotlib/axes/_axes.py

```diff
@@ -2950,6 +2950,8 @@ def errorbar(self, x, y, yerr=None, xerr=None,
                 upper errors.
```

Comment:
I think I'd rather have it go as a parenthetical statement here? e.g. `(both of them must be positive)`.  (It's also true for the other cases but I think it's kind of obvious for them?)